src/main/java/org/uwdigi/rag/config/AppConfig.java

```diff
@@ -99,6 +106,16 @@ public AppConfig(FhirDbConfig fhirDbConfig) {
     this.fhirDbConfig = fhirDbConfig;
   }
 
+  @Bean
+  public PromptTemplate sqlPromptTemplate() {
+    return sqlPromptTemplate;
+  }
+
+  @Bean
+  public Boolean useCloudLLMOnly() {
```

Comment:
Minor things, but there's no good reason to use `Boolean` here instead of `boolean`.